lib/inject.js

```diff
@@ -86,57 +86,70 @@ function moveModuleUp(source, target, module) {
  * Inject requirements into packaged application.
  * @return {Promise} the combined promise for requirements injection.
  */
-function injectAllRequirements(funcArtifact) {
+async function injectAllRequirements(funcArtifact) {
   if (this.options.layer) {
     // The requirements will be placed in a Layer, so just resolve
     return BbPromise.resolve();
   }
 
-  this.serverless.cli.log('Injecting required Python packages to package...');
+  let injectProgress;
+  if (this.progress && this.log) {
+    injectProgress = this.progress.get('python-inject-requirements');
+    injectProgress.update('Injecting required Python packages to package');
+    this.log.info('Injecting required Python packages to package');
+  } else {
+    this.serverless.cli.log('Injecting required Python packages to package...');
+  }
 
-  if (this.serverless.service.package.individually) {
-    return BbPromise.resolve(this.targetFuncs)
-      .filter((func) =>
-        (func.runtime || this.serverless.service.provider.runtime).match(
-          /^python.*/
+  try {
+    if (this.serverless.service.package.individually) {
+      await BbPromise.resolve(this.targetFuncs)
```

Comment:
It should `return await` to work in same way (but not sure, maybe it won't make difference)

Reply:
I've skipped that return because the result is not actually used anywhere